src/core/adapters/linear.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Linear adapter
+ *
+ * The adapter extracts ticket information from the markup. Note that linear
+ * obfuscates classes, so the DOM selectors are probably not reliable (for one,
+ * they are locale dependent, but at this time Linear is only available in
+ * English).
+ *
+ */
+
+import { match } from "micro-match";
+
+import type { TicketData } from "../types";
+import { $text } from "./dom-helpers";
+
+async function scan(url: URL, document: Document): Promise<TicketData[]> {
+  if (url.hostname !== "linear.app") return [];
+
+  const { id, issueType } = match("/:team/:issueType/:id/:slug", url.pathname);
+  const title = $text('[title="Edit issue title"]', document);
+
+  if (!id) return [];
+
+  const ticket = {
```

Comment:
Is there also a `description` field in a somewhat commit-message-body-usable format?

Reply:
If we use the API (see my [other comment](https://github.com/bitcrowd/tickety-tick/pull/336#discussion_r820795967)) then yes, otherwise, probably not very reliably.